src/Middleware/HttpLogging/test/W3CLoggingMiddlewareTests.cs

```diff
@@ -103,6 +103,48 @@ public async Task DefaultDoesNotLogOptionalFields()
         Assert.DoesNotContain(lines[3], "Snickerdoodle");
     }
 
+    [Fact]
```

Comment:
Add a test for if the user passes an invalid header name (once we decide on a behavior for that)

Reply:
Would still like a test that confirms that passing an invalid header doesn't cause the middleware to throw, and that the log file looks how we expect

Reply:
Added here a792f7ece52e774bc84b69f59f36bb3702a675d6